app/pages/login/login.html

```diff
@@ -6,23 +6,25 @@
 
 <ion-content>
 
-  <form (submit)="submitLogin(username, password)">
+  <form [ngFormModel]="loginForm" (ngSubmit)="submitLogin(loginForm.value.username, loginForm.value.password)">
```

Comment:
Let's declare a local variable here.
````html
<form #form>
```
This would allow us, to drop the `FormBuilder` code, because then you can just assign the controls to the form.

```html
<input ngControl="username" required>
```

http://plnkr.co/edit/ChQUSZ7EhsBtJcdXGj1e?p=preview
https://angular.io/docs/ts/latest/guide/forms.html